lib/compiler.js

```diff
@@ -136,10 +141,11 @@ function assertIsValidModelConfig(config) {
  * @private
  */
 
-function findScripts(dir) {
+function findScripts(dir, extensions) {
   assert(dir, 'cannot require directory contents without directory name');
 
   var files = tryReadDir(dir);
+  extensions = [].concat(extensions || _.keys(require.extensions));
```

Comment:
Hmm, is this the same as the following simpler form? `extensions = extensions || _.keys(require.extensions)`?